sources/common/infoschema.go

```diff
@@ -253,3 +254,14 @@ func (is *InfoSchemaImpl) GetIncludedSrcTablesFromConv(conv *internal.Conv) (sch
 	}
 	return schemaToTablesMap, nil
 }
+
+// SanitizeDefaultValue removes extra characters added to Default Value in information schema in MySQL.
```

Comment:
This code is added in another PR for review and merging, the testing for it is added there. Hence its missing here:
https://github.com/GoogleCloudPlatform/spanner-migration-tool/pull/963